idaes_examples/nb/properties/custom/custom_physical_property_packages_src.ipynb

```diff
@@ -67,7 +67,7 @@
     "\n",
     "An important aspect of the IDAES Core Modeling Framework is that a modeler only needs to provide calculations for those properties that they will use within their process. Put another way, modelers do not need to include calculations for a property that they are not going to use in their model – this allows modelers to avoid introducing unnecessary complexity into their models to calculate a property they do not actually need. When combined with flexibility elsewhere in the modeling framework to control which equations are written in the unit models, this can even allow users to avoid calculating properties that would normally be considered mandatory – for example, a property package for a conceptual design flowsheet which does not include energy or momentum balances would not need to define specific enthalpy or even temperature and pressure as these will not be required by the unit models.\n",
     "\n",
-    "This then raises the question of how do you know what properties you will need, especially if you are using models from a library you did not write yourself. To answer this, you should refer to the model documentation, and you can also use the [IDAES Properties Interrogator tool](https://idaes-pse.readthedocs.io/en/latest/technical_specs/model_libraries/generic/property_models/interrogator.html) to analyze your flowsheet and determine what properties are required.\n",
+    "This then raises the question of how do you know what properties you will need, especially if you are using models from a library you did not write yourself. To answer this, you should refer to the model documentation, and you can also use the [IDAES Properties Interrogator tool](https://idaes-pse.readthedocs.io/en/latest/reference_guides/model_libraries/generic/property_models/interrogator.html) to analyze your flowsheet and determine what properties are required.\n",
```

Comment:
Should we be using `latest` or `stable` in all of these links? I would have thought `stable` was generally the better option.

It looks like we were generally using `latest` in the past however. Either way, we should make sure we are consistent.

Reply:
The use of `latest` vs. `stable` was inconsistent when I picked this up. I agree that being consistent in this regard is a good idea - the only reason I didn't do that already was that I didn't know if I should prefer `latest` or `stable`.

@dangunter would you prefer `latest` or `stable` in the links?

Reply:
I went ahead and replaced all instances of `latest` with `stable`, both for IDAES and Pyomo readthedocs links.

Reply:
Since we have a pre-processing step that does arbitrary things, we could also consider using that to fix links or even embed "macros" like `{{docs}}` that will get expanded into whatever we want in the Jupyterbook and tutorial/exercise/"user" versions of the notebooks.